docs/specification.md

```diff
@@ -57,26 +57,162 @@ A2A revolves around several key concepts. For detailed explanations, please refe
 
 ## 3. Transport and Format
 
-### 3.1. Transport Protocol
+### 3.1. Transport Layer Requirements
+
+A2A supports multiple transport protocols, all operating over **HTTP(S)**:
 
 - A2A communication **MUST** occur over **HTTP(S)**.
-- The A2A Server exposes its service at a URL defined in its `AgentCard`.
+- The A2A Server exposes its service at one or more URLs defined in its `AgentCard`.
+
+### 3.2. Supported Transport Protocols
+
+A2A defines three transport protocols:
 
-### 3.2. Data Format
+#### 3.2.1. JSON-RPC 2.0 Transport (Mandatory)
 
-A2A uses **[JSON-RPC 2.0](https://www.jsonrpc.org/specification)** as the payload format for all requests and responses (excluding the SSE stream wrapper).
+**All A2A-compliant agents MUST support JSON-RPC 2.0 transport.** This is the baseline transport for A2A interoperability.
 
+- The primary data format is **[JSON-RPC 2.0](https://www.jsonrpc.org/specification)** for all requests and responses (excluding SSE stream wrapper).
 - Client requests and server responses **MUST** adhere to the JSON-RPC 2.0 specification.
-- The `Content-Type` header for HTTP requests and responses containing JSON-RPC payloads **MUST** be `application/json`.
+- The `Content-Type` header for HTTP requests and responses **MUST** be `application/json`.
```

Comment:
Doesn't this change prevent streaming responses from using text/event-stream as the response content type?

Reply:
The original Line 70 says "containing JSON-RPC payloads **MUST** be `application/json`"
The new line 77 removes "containing JSON-RPC payloads" which, in my opinion, causes the statement to be not true when using JSON-RPC transport and getting a response from "message/stream".